pkg/util/cloudprovider.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package util
```

Comment:
any reason not to put this in a more meaningful package? (`pkg/metadata` for example maybe?) :)

Reply:
I think cloudprovider logic might be use by other agent/package. For not it's pretty inventory oriented but the more we had to it the more useful it will be to have it in util.